tests/python/frontend/onnx/test_forward.py

```diff
@@ -5241,23 +5241,7 @@ def verify_eyelike(indata, dynamic=False):
     "test_training_dropout_mask",
     "test_training_dropout_zero_ratio",
     "test_training_dropout_zero_ratio_mask",
-    "test_tril",
-    "test_tril_pos",
-    "test_tril_square",
-    "test_tril_square_neg",
-    "test_tril_neg",
-    "test_tril_one_row_neg",
-    "test_tril_out_neg",
-    "test_tril_out_pos",
     "test_tril_zero",
```

Comment:
I haven't looked at this op at all. How tricky would it be to support the zero case? Otherwise LGTM.

Reply:
It actually works on llvm and cuda. I was testing on my macbook and it seems like the metal backend in general doesnt support empty tensors. I think for CI we could add these cases.

Reply:
Seems like it also doesnt work with nvptx for the same issue with empty tensors.